client/src/contexts/UserContext.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react'
+const UserContext=React.createContext({});
+
+export function UserStore(props) {
+    const [user, setUser] = useState(false)
+
+    const handleLogin = (email, password) => {
+        if (true) {
+            fetch("/auth/login", {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json'
+                },
+                body: JSON.stringify({
+                    'email': email,
+                    'password': password,
+                })})
+                .then(response => response.json())
+                .then((response) => {
+                    console.log(response)
+                    if (response.status === 'success') {
+                        console.log('Success:', email);
+                        setUser(true);
+                        console.log(user)
+                    }
+                    else {
+                        window.alert(response.message);
+                        console.log(response.message);
+                    }
+                })
+                .catch((error) => {
+                    console.error('Error:', error)
+                })
+            }
+      }
+      const handleLogout = () => {
+        setUser(false);
```

Comment:
We'll eventually need to make an API call here in handleLogout, to clear the cookies.

We also need a handleSignup function as well.